src/containers/OrderUpdate.vue

```diff
@@ -75,6 +79,90 @@ export default {
       } catch (err) {
         this.handleError(err);
       }
+    },
+
+    onPrint() {
+      if (!this.params) return [];
+
+      const { categories, squares, parts } = this.params;
+
+      var dd = {
+        content: [
+          {
+            stack: [
+              this.$t("score"),
+              {
+                text: `${this.order.date}`,
+                style: "subheader"
+              }
+            ],
+            style: "header"
+          },
+          {
+            text: [
+              `${this.$t("clientName")}: ${this.order.clientName}\n`,
+              `${this.$t("phoneNumber")}: ${this.order.phoneNumber}\n`,
+              `${this.$t("carModel")}: ${this.order.make}\n`,
+              `${this.$t("model")}: ${this.order.model}\n`,
+              `${this.$t("carNumber")}: ${this.order.carNumber}\n`
+            ],
+            style: "text"
+          },
+          {
+            table: {
+              widths: ["*", 125, 75, 75],
+              body: [
+                [
+                  { text: `${this.$t("part")}`, style: "tableHeader" },
```

Comment:
just
{ text: this.$t("part"), style: "tableHeader" }

Reply:
agree